website/content/en/pre-docs/tasks/running-pods.md

```diff
@@ -0,0 +1,222 @@
+The next layer comes from you adding specifications to your Kubernetes pod deployments.
+When you request constraints, those constraints must fall within the provisioner constraints or the pods will not deploy.
+For example, if the provisioner sets limits that allow only a particular zone to be used, and a pod asks for a different zone, deployment will fail.
+
+Constraints you can request include:
+
+* **Resource requests**: Request that certain amount of memory or CPU be available.
+* **Disruption budget**: Limit the disruptions that an application can experience.
+* **Node selection**: Choose to run on a node that is has a particular label (`nodeSelector`) or name (`nodeName`).
+* **Topology spread**: Use topology spread to help insure availability of the application.
+* **Node affinity**: Draws a pod to run on nodes with particular attributes (affinity) or that don't have those attributes (antiaffinity).
+* **Persistent volumes**: Insures the availability of selected persistent volumes.
+
+The constraints that Karpenter supports are based mostly on features built into Kubernetes and a few that are specific to the cloud provider.
+In fact, you the exact same constraints can be used on clusters by Cluster Autoscaler or even just by the Kubernetes scheduler to pre-provision capacity.
+Keep in mind, however, that not all Kubernetes constraints are supported or recommended with Karpenter.
+This section steps through examples of those that are supported.
+
+{{% alert title="Note" color="primary" %}}
+The constraints you identify through affinity and node selection include a subset of Kubernetes [Well-Known Labels, Annotations and Taints](Well-Known Labels, Annotations and Taints).
+See [Specifying Values to Control AWS Provisioning](/docs/cloud-providers/aws/aws-spec-fields) for a listing and descriptions of those values.
+{{% /alert %}}
+
+## Resource requests (`resources`)
+
+Within a Pod spec, you can both make requests and set limits on resources a pod needs, such as CPU and memory.
+For example:
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp
+spec:
+  containers:
+  - name: app
+    image: myimage
+    resources:
+      requests:
+        memory: "128Mi"
+        cpu: "500m"
+      limits:
+        memory: "256Mi"
+        cpu: "1000m"
+```
+In this example, the container is requesting 128MiB of memory and .5 CPU.
+Its limits are set to 256MiB of memory and 1 CPU.
+
+
+See [Managing Resources for Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) for details on resource types supported by Kubernetes, [Specify a memory request and a memory limit](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/#specify-a-memory-request-and-a-memory-limit) for examples of memory requests, and [Specifying Values to Control AWS Provisioning](/docs/cloud-providers/aws/aws-spec-fields) for a list of supported resources.
```

Comment:
Another good section.

Reply:
I realize this may be duplicitous with kubernetes docs. In time, we can learn if this info is helpful to karpenter users.